setup.py

```diff
@@ -99,7 +100,8 @@
     author="Allen Institute for Artificial Intelligence",
     author_email="allennlp@allenai.org",
     license="Apache",
-    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"])
+    + find_namespace_packages(include=["allennlp_plugins.*"]),
```

Comment:
Note that having the folder `allennlp_plugins` in this project is not necessary. However, I added it so doing `import allennlp_plugins` in `allennlp/commons/util.py` doesn't fail if there are no plugins, and to avoid to do `try`-`except ImportError`, etc. And so it's included in the pip package, I added this line.

And I added the `.keep` file under `allennlp_plugins` so the folder isn't empty and can be added to git.